content/applications/finance/accounting/bank/feeds/enablebanking.rst

```diff
@@ -0,0 +1,78 @@
+===============================================
+Enable Banking as bank synchronization provider
+===============================================
+
+**Enable Banking** is a third-party provider that fetches account transactions
+from your bank accounts. It supports ~350 institutions in more than 20 countries
```

Comment:
Oh well, to me it's still the same idea. @LoredanaLrpz 

It's true we, French speakers, tend to add too many different ideas in one paragraph, usually but that's not the case here.

Here the idea is "a short and user-oriented explanation of what **Enable Banking** is".

Two different ideas that would need to be split would be, for example:
- Third-party providers require a login, password and secret key
- To retrieve them, go to your dashboard blablabla

However, great catch to put the "free" here rather than having an additional paragraph later just for that.